### ISSUE TITLE
embed_md.py cannot pull several fields

### ISSUE THREAD

**Issue by hocksteindw** [brnco/avmpi_scripts#4]
Hi! Seems like a few issues are hangin' out in the code here. Ideally, we'd be able to just insert nothing if the field is not present. Seems like a few formatting str/int things are causing problems.. The filename and the record seem to match, the record is filled in appropriately, and things still seem to panic when it tries to pull the "equipment used-digitization" field. Looking at Airtable, is that field there as text?:

```
(venv) audio_transfer@SLA-CG-3118A avmpi_scripts % python3 embed_md.py -dadir /Users/audio_transfer/Desktop/TransferSession_DateLogs/20240816_1/Audio -daid sinaa_levy_sr_08_01.wav
starting...
2024-08-19 09:19:48 INFO: initializing logs...
2024-08-19 09:19:48 INFO: preparing to embed metadata into wave files...
2024-08-19 09:19:48 INFO: searching Airtable table: <Table base='appU0Fh8L9xVZBeok' name='Digital Assets'>
2024-08-19 09:19:48 INFO: for value: sinaa_levy_sr_08_01.wav
2024-08-19 09:19:48 INFO: in field: Digital Asset ID
2024-08-19 09:19:49 INFO: searching Airtable table: <Table base='appU0Fh8L9xVZBeok' name='Digital Assets'>
2024-08-19 09:19:49 INFO: for value: sinaa_levy_sr_08_01.wav
2024-08-19 09:19:49 INFO: in field: Digital Asset ID
2024-08-19 09:19:49 INFO: searching Airtable table: <Table base='appU0Fh8L9xVZBeok' name='Digital Assets'>
2024-08-19 09:19:49 INFO: for value: sinaa_levy_sr_08_01.wav
2024-08-19 09:19:49 INFO: in field: Digital Asset ID
Traceback (most recent call last):
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/files.py", line 228, in from_atbl
    value = atbl_rec_digital_asset['fields'][mapping['atbl']]
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
KeyError: 'Coding History'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/files.py", line 50, in from_atbl
    value = atbl_rec_digital_asset['fields'][mapping['atbl']['name']]
                                             ~~~~~~~~~~~~~~~^^^^^^^^
TypeError: string indices must be integers, not 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/files.py", line 58, in from_atbl
    value = atbl_rec_digital_asset['fields'][mapping['atbl']]
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
KeyError: 'Equipment Used - Digitization (BWF)'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/embed_md.py", line 154, in <module>
    main()
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/embed_md.py", line 150, in main
    embed_metadata(kwvars)
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/embed_md.py", line 78, in embed_metadata
    bwf = files.BroadcastWaveFile().from_atbl(kwvars['daid'])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/files.py", line 231, in from_atbl
    bwf_codhist = CodingHistory().from_atbl(digital_asset_id)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/audio_transfer/Code/avmpi_scripts/avmpi_scripts/files.py", line 61, in from_atbl
    raise RuntimeError(f"not enough info to create Coding History for this asset. {details}")
RuntimeError: not enough info to create Coding History for this asset. Was this created with the old workflow? Exiting...
```

**Comment by brnco** [comment]
Dan, I believe this was a result of a mismatch between a field name in Airtable and the field name in the `field_mappings.json` config. I have updated the field mapping in Airtable with the (BWF) suffix we use there and had success on my end. Give this file a shot again tomorrow.

**Comment by brnco** [comment]
I ended up re-doing a bunch of this and formulating the coding history in Airtable, instead (per our emails).  So, now the BWF object inherits that info directly from Airtable instead of building it itself, within the script. That process is what originated this error, so I'm hopeful it'll work better now.

**Comment by hocksteindw** [comment]
Yay! The script works! However, I have some small things to note that should be simple fixes. I'll open another issue, but you can close this one! Thanks!

**Comment by brnco** [comment]
fixed in [fcda8da](https://github.com/brnco/avmpi_scripts/commit/fcda8da733ca36c748a8de467ac4c29676abb185)
